cred-manager/src/sourcecred/service/sourcecred.service.ts

```diff
@@ -111,6 +110,26 @@ export class SourceCredService {
     return success;
   }
 
+  async resetSourceCred() {
+    try {
+      await executeCommand(`cd ${this.sourceCredPath} && yarn clean-all`);
+      await executeCommand(`rm -rf ${this.sourceCredPath}/data/ledger.json`);
```

Comment:
AFAIK this could break the whole command chain if the file does not exist anymore, and as the resetSourceCred function is called both before and after, this could cause problems.
Is this assumption correct or am I missing something here?